optuna/integration/catalyst.py

```diff
@@ -16,6 +16,10 @@
 class CatalystPruningCallback(Callback):
     """Catalyst callback to prune unpromising trials.
 
+    See `the example <https://github.com/optuna/optuna/blob/master/
+    examples/catalyst_simple.py>`_ if you want to add a pruning callback
+    which observes accuracy of a Catalyst's SupervisedRunner.
```

Comment:
Some nits. You might have reused the sentence from other examples, though I'm not sure they're 100% grammatically correct neither...
```suggestion
    which observes the accuracy of Catalyst's ``SupervisedRunner``.
```